cclib/bridge/cclib2horton.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Bridge for using cclib data in Horton (http://theochem.github.io/horton)."""
+
+import numpy
+from cclib.parser.data import ccData
+from cclib.parser.utils import find_package
+
+_old_horton = False
+
+_found_horton = find_package('horton')
+if _found_horton:
+    try:
+        from horton import __version__
+    except:
+        _old_horton = True
+    if __version__[0] == "2":
+        # Horton 2
+        from horton.io.iodata import IOData
+
+_found_iodata = find_package('iodata')
+if _found_iodata:
+    from iodata import IOData
+    from iodata.orbitals import MolecularOrbitals
+    from iodata.basis import MolecularBasis
+
+def _check_horton(found_horton, found_iodata, old_horton):
+    if old_horton:
+        raise ImportError("You must have at least version 2 of `horton` to use this function.")
+    elif not found_horton and not found_iodata:
+        raise ImportError("You must install `horton` to use this function.")
+    if (found_iodata):
+        return 3
+    elif (found_horton):
+        return 2
+
+
+def makehorton(ccdat):
+    """ Create horton IOData object from ccData object """
+
+    _check_horton(_found_horton, _found_iodata, _old_horton)
```

Comment:
Don't these variables live at module level? You don't need to pass them in as arguments.

Reply:
Yes those variables do live at module level. I've wrote the function this way to be consistent with other bridges, but I can modify this to make it more readable and efficient. Thanks!

Reply:
Alright, thanks for being consistent.